package.json

```diff
@@ -27,7 +27,8 @@
     "build": "npm run clean && babel src -d dist --copy-files && BABEL_ENV=es babel src -d es --copy-files && node-sass src/main.scss dist/style.css --output-style compressed && npm run build:browser",
     "lint": "eslint src tests showcase docs --ignore-pattern node_modules --ignore-pattern bundle.js",
     "lint-styles": "stylelint src/styles/*.scss --syntax scss",
-    "test": "node --inspect node_modules/.bin/babel-node tests/index.js",
+    "test:windows": "babel-node --inspect ./tests/index.js",
```

Comment:
remove?

Reply:
i wasn't intended to remove test command just one i created for windows, not sure how this happen,  i added my own because i cant "npm run test" on windows, i have error ( this one https://github.com/jmcriffey/babel-istanbul/issues/70 ) , same goes for lint i should add "linebreak-style": ["error", "windows"]" to make it work, probably tests wasn't meant to run on windows platform

Reply:
that's super weird! i don't really know what to do about it. Maybe file a bug and it can be looked into separately?

Reply:
ok i will create separate issue for this